specification/maps/data-plane/Search/preview/2023-04-01-preview/search.json

```diff
@@ -0,0 +1,1427 @@
+              "x-ms-request-id": {
+                "description": "The autosuggest service request ID.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "UsageType": {
+      "type": "string",
+      "enum": [
+        "Display",
+        "Route"
+      ],
+      "x-ms-enum": {
+        "name": "UsageTypeEnum",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "Display"
+          },
+          {
+            "value": "Route"
+          }
+        ]
+      }
+    },
+    "AutosuggestResponse": {
+      "description": "This object is returned from a successful autosuggest call",
+      "type": "object",
+      "properties": {
+        "type": {
+          "$ref": "#/definitions/FeatureCollectionType"
+        },
+        "features": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AutosuggestFeature"
+          }
+        },
+        "nextLink": {
+          "$ref": "#/definitions/NextLink"
+        }
+      }
+    },
+    "AutosuggestFeature": {
+      "description": "`GeoJSON GeocodingFeature` object",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../Common/preview/2022-09-01-preview/common.json#/definitions/GeoJsonFeature"
+        },
+        {
+          "$ref": "#/definitions/AutosuggestProperties"
+        }
+      ]
+    },
+    "AutosuggestProperties": {
+      "description": "Properties of Autosuggest Result",
+      "type": "object",
+      "properties": {
+        "type": {
+          "description": "Place or Address",
+          "type": "string",
+          "enum": [
+            "Place",
+            "Address"
+          ],
+          "x-ms-enum": {
+            "name": "AutosuggestResultType",
+            "modelAsString": true,
+            "values": [
+              {
+                "name": "Place",
+                "value": "Place"
+              },
+              {
+                "name": "Address",
+                "value": "Address"
+              }
+            ]
+          }
+        },
+        "address": {
+          "description": "Address",
+          "type": "object",
+          "allOf": [
+            {
+              "$ref": "#/definitions/Address"
+            },
+            {
+              "type": "object",
+              "properties": {
+                "houseNumber": {
+                  "description": "The house number of the address.",
+                  "type": "string"
+                },
+                "streetNumber": {
+                  "description": "The street number of the address.",
+                  "type": "string"
+                }
+              }
+            }
+          ]
+        },
+        "name": {
+          "description": "The name of the place.",
```

Comment:
Can we also remind user this field will not show if the returned type is `Address`?